### PR TITLE
Additional VSCode Telemetry

### PR DESCRIPTION
**Pull request by gj**

### CODE REVIEW

File: vscode/oso/client/src/telemetry.ts [hunk 1]
```diff
@@ -68,34 +68,51 @@ export function flushQueue(
...
+        log.appendLine(e);
       }
     })();
   };
 }
 
 export type TelemetryEvent = {
   diagnostics: Diagnostic[];
-  has_resource_blocks: boolean;
+  general_stats: {
```
Comment (dhatch) [inline vscode/oso/client/src/telemetry.ts:91]: cool cool

File: vscode/oso/client/src/telemetry.ts [hunk 2]
```diff
@@ -107,19 +124,22 @@ export function enqueueEvent(
       const warnings = diagnostics.filter(d => d.severity === Severity.Warning);
 
       const loadEvent: MixpanelEvent = {
-        event: 'load',
+        event: 'TEST_load',
```
Comment (dhatch) [inline vscode/oso/client/src/telemetry.ts:127]: fix before merging

File: polar-language-server/src/lib.rs [hunk 3]
```diff
@@ -277,7 +269,114 @@ impl PolarLanguageServer {
...
+            diagnostics,
+            general_stats: GeneralStats {
+                polar_chars,
+                polar_files: self.documents.len(),
+                ..Default::default()
+            },
+            ..Default::default()
+        };
+
+        let lines = self.documents.values();
```
[the remark this answers is not in the document]
Comment (gj) [reply polar-language-server/src/lib.rs:320]: Thought about that but the tradeoff is making the core aware of things that only matter for PLS

File: vscode/oso/client/src/telemetry.ts [hunk 4]
```diff
@@ -21,18 +21,21 @@ const MIXPANEL_PROJECT_TOKEN = 'd14a9580b894059dffd19437b7ddd7be';
 const mixpanel = Mixpanel.init(MIXPANEL_PROJECT_TOKEN, { protocol: 'https' });
 
 type MixpanelLoadEvent = {
-  event: 'load';
+  event: 'TEST_load';
```
[the remark this answers is not in the document]
Comment (gj) [reply vscode/oso/client/src/telemetry.ts:24]: Will fix before releasing; want to keep it like this until we release so we don't pollute the "real" event key on mixpanel

[the remark this answers is not in the document]
Comment (dhatch) [reply vscode/oso/client/src/telemetry.ts:24]: ok

**Review by dhatch — APPROVED** [review]

**Pull request merged**